edxlearndot/learndot.py

```diff
@@ -166,7 +199,7 @@ def is_valid(cls, status):
         return hasattr(cls, status) and getattr(cls, status) == status
 
 
-class LearndotAPIClient(object):
+class LearndotAPIClient(object):  # pylint: disable=useless-object-inheritance
```

Comment:
nit: python3 classes all inherit from object:
```suggestion
class LearndotAPIClient: